posthog/api/test/test_insight_funnels.py

```diff
@@ -1,71 +1,14 @@
-import json
 from datetime import datetime
 from typing import Any, Dict, List, Union
-from uuid import uuid4
 
 from django.test.client import Client
 from rest_framework import status
 
-from ee.api.test.base import LicensedTestMixin
-from ee.clickhouse.models.event import create_event
 from ee.clickhouse.test.test_journeys import journeys_for
 from ee.clickhouse.util import ClickhouseTestMixin
-from ee.models.explicit_team_membership import ExplicitTeamMembership
-from posthog.api.test.test_insight import insight_test_factory
-from posthog.models.organization import OrganizationMembership
-from posthog.models.person import Person
 from posthog.test.base import APIBaseTest
 
 
-def _create_person(**kwargs):
-    person = Person.objects.create(**kwargs)
-    return Person(id=str(person.uuid))
-
-
-def _create_event(**kwargs):
-    kwargs.update({"event_uuid": uuid4()})
-    create_event(**kwargs)
-
-
-class ClickhouseTestInsights(
-    ClickhouseTestMixin, LicensedTestMixin, insight_test_factory(_create_event, _create_person)  # type: ignore
-):
-    # Extra permissioning tests here
-    def test_insight_trends_allowed_if_project_open_and_org_member(self):
```

Comment:
Are these tests still relevant, can't see them in the new code but might be missing something.

Reply:
it's in this file now `posthog/api/insight.py`